### ISSUE TITLE
Separate install folder

### ISSUE THREAD

**Issue by prhost** [copona/copona#134]
I find it interesting to create a separate installer in another repository, for the sake of security, organization, and performance. Also make the installer download stable releases, along with all dependencies, also make the installer perform all migrations. We can also design a new layout, start to give more appearance of the copona, and remove that appearance of opencart, make it easier and more practical to install, show installation progress progressbar, remove maximind and opencart extensions, support all drivers that ORM supports, well, I know it's not a priority right now, but some modifications I'm doing in the configuration part of the copona, I need to modify it too, so I got this idea.

Using https://deployer.org would be a good way to facilitate the installer's development.

**Comment by arnisjuraga** [comment]
It sounds like a huge dev.
Although, installation process itself is not so complicated. But there are some compatibility checks, "update" checks and so.

For future sake, it could be as an alternate install process we can test.
Just try to make it simple as possible, to not over complicate included libraries.

**Comment by stefanhuber** [comment]
To be honest I have never used the Web based installer of opencart, as the CLI installation works great ;)

I am using Robo Taskrunner to install opencart from cli, with all db generation and additional things.
https://github.com/beyondit/opencart-project-template/blob/master/RoboFile.php#L58

**Comment by arnisjuraga** [comment]
This looks great! Copona has the old Opencart ```install_cli.php``` file, wchich we have fixed and tested as good enough. Your Robo installs Opencart from Composer? With a first release, Copona can be added to Composer and in that case (never did that 👍  !)?

**Comment by prhost** [comment]
@stefanhuber convert html to pdf with TCPDF does not get heavy? I've had some problems like this, I just can not remember which library I used.

**Comment by prhost** [comment]
Sorry, I ended up commenting on the wrong issue, and I accidentally deleted the comment that was last:

@arnisjuraga had said:
This looks great! Copona has the old Opencart
```install_cli.php```
file, wchich we have fixed and tested as good enough. Your Robo installs Opencart from Composer? With a first release, Copona can be added to Composer and in that case (never did that :+1:  !)?

**Comment by prhost** [comment]
@stefanhuber would be nice to do something like this but develop a cli itself that supports several commands, one of them the setup of copona, using `symfony/console` would be a good way.

@arnisjuraga has how to add yes, I even added the copona in pakagist to do some package installation and project creation tests. We can put some post-scripts to run after the creation of the project via composer and call the setup similar to @stefanhuber, that would be great.

**Comment by stefanhuber** [comment]
@prhost yes some time ago i started with a project (https://github.com/beyondit/ocok) which is based on symfony.

There is even one thing, which might be interesting to think about together: (https://github.com/beyondit/ocok/blob/codegenerator/src/BeyondIT/OCOK/GeneratorCommand.php). It is a template-based codegenerator for the opencart admin. For extensions this is quite helpful, as one can fully generate the admin part...

I think a CLI would be quite helpful, especially for developers!

**Comment by arnisjuraga** [comment]
@stefanhuber both things sounds interesting to me!
